src/main/cpp/t06_longest.cpp

```diff
@@ -22,5 +22,22 @@
 using namespace std;
 
 int t06_longest() {
+	int pos = 0;
+    string s, maxs = "";
+    getline(cin, s);
 
+    while((int)s.find(" ", pos) > -1){
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Inefficient usage of string::find() in condition; string::starts_with() could be faster.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4513044)